### DISCUSSION TITLE
Google Fit sync questions

### DISCUSSION THREAD

**Discussion by dagord** [NightscoutFoundation/xDrip#3780] [General]
Hello,
I've some doubts about the Google Fit syncronization that I'm testing in the last few days.
- Even if I've set the glucose units to mg/dL, Google Fit shows the readings in the mmol/L format: I don't know whetever it's an issue on xDrip or on Google Fit side
- I added some fake food/carbs data into Google Fit but xDrip doesn't get them (even if I enabled the get/send get data to/from Health Connect options); I don't know if it's a bug or a feature yet to be implemented, which I think would be very useful when using third-part food tracking apps

**Comment by PhilTU** [comment]
Google Fit only works in mmol/l
